ui/lib/apps/ClusterInfo/components/StoreLocationTree/index.tsx

```diff
@@ -93,6 +282,9 @@ export default function StoreLocationTree({
           `translate(${t.x + margin.left}, ${t.y + margin.top}) scale(${t.k})`
         )
 
+        // don't show tooltip if zoom or drag
+        enableTooltip.current = t.x === 0 && t.y === 0 && t.k === 1
```

Comment:
Can this be put in `zoom.on('end')` without expressions?

Reply:
Good catch! let me try it.